### DISCUSSION TITLE
Styling - TextBox Selected BorderBrush and BorderThickness

### DISCUSSION THREAD

**Discussion by marcosgerene** [AvaloniaUI/Avalonia#13054] [Q&A] [answered]
Hi folks,

I need some help in styles.

![image](https://github.com/AvaloniaUI/Avalonia/assets/15945003/8e16a744-b44e-481c-8b20-25845f3b52ae)
![image](https://github.com/AvaloniaUI/Avalonia/assets/15945003/3780a9e8-b154-4a96-b2a3-49ab6804f055)

My desired behavior: when textbox selected the bordercolor is #455a64 and BorderThicknes "0 0 0 1"

----

Code:
``` 
<WrapPanel>
	<StackPanel Classes="Padrao">
		<TextBlock Classes="Padrao" Text="Id"/>
		<TextBox Classes="Padrao" Text="100" />
	</StackPanel>

	<StackPanel Classes="Padrao">
		<TextBlock Classes="Padrao" Text="Nome" />
		<TextBox Classes="Padrao" Text="FAIXA LATERAL BL BLA BKA BA A" />
	</StackPanel>
</WrapPanel>
```

Styles:
```
<!-- #region Campos -->
<Style Selector="StackPanel.Padrao">
	<Setter Property="Margin" Value="0 0 10 0"/>
</Style>

<Style Selector="TextBlock.Padrao">
	<Setter Property="HorizontalAlignment" Value="Left"/>
	<Setter Property="FontWeight" Value="Normal"/>
</Style>

<Style Selector="TextBox.Padrao">
	<Setter Property="Padding" Value="5 10 0 0"/>
	<Setter Property="BorderThickness" Value="0 0 0 1"/>
	<Setter Property="CornerRadius" Value="0 0 0 0"/>		
</Style>	
<!-- #endregion Campos -->
```

I am using Avalonia on 11.0.4 version.

Thank you very much

**Comment by timunie** [comment]
check original styles for reference. 

Note: Probably also check what FluentAvalonia does. they do underline the text and ship probably the style you are looking for.

**Comment by marcosgerene** [comment, marked as answer]
![image](https://github.com/AvaloniaUI/Avalonia/assets/15945003/72149080-f57a-4069-87af-bfe9327dd260)

```
<Style Selector="TextBox.Padrao:focus /template/ Border#PART_BorderElement">
	<Setter Property="BorderBrush" Value="#455a64"/>
	<Setter Property="BorderThickness" Value="0 0 0 3"/>
</Style>
```

I got it on Bing Chat it worked as expected.
Thank you very much @timunie
